packs/data/feats.db/slayers-presence.json

```
@@ -42,7 +42,7 @@
                     }
                 ],
                 "selector": "skill-check",
-                "text": "@Localize[PF2E.SpecificRule.UndeadSlayer.SlayersPresence.Note",
+                "text": "@Localize[PF2E.SpecificRule.UndeadSlayer.SlayersPresence.Note]",
```

```suggestion
                "text": "PF2E.SpecificRule.UndeadSlayer.SlayersPresence.Note",
```
We can skip Localize calls entirely if it's the only text in a note RE

---

Done.